packages/vats/test/test-provisionPool.js

```diff
@@ -1,29 +1,28 @@
 // @ts-check
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { test as unknownTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
-import path from 'path';
-import { unsafeMakeBundleCache } from '@agoric/swingset-vat/tools/bundleTool.js';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import { CONTRACT_ELECTORATE, ParamTypes } from '@agoric/governance';
+import committeeBundle from '@agoric/governance/bundles/bundle-committee.js';
 import {
   makeMockChainStorageRoot,
   setUpZoeForTest,
   withAmountUtils,
 } from '@agoric/inter-protocol/test/supports.js';
-import committeeBundle from '@agoric/governance/bundles/bundle-committee.js';
-import { AmountMath, makeIssuerKit } from '@agoric/ertp';
-import { E, Far } from '@endo/far';
+import { WalletName } from '@agoric/internal';
```

Comment:
Was `WalletName` moved or deleted? I'm having trouble reading this diff.

Reply:
the imports were auto cleaned up by VSCode. So just reording